requirements.txt

```diff
@@ -60,19 +60,20 @@ fido2==0.9.3
 flask==2.1.2
     # via
     #   -r requirements.in
+    #   flask-basicauth
     #   flask-login
     #   flask-redis
     #   flask-wtf
     #   gds-metrics
     #   notifications-utils
+flask-basicauth==0.2.0
+    # via -r requirements.in
 flask-login==0.6.1
     # via -r requirements.in
 flask-redis==0.4.0
     # via notifications-utils
 flask-wtf==1.0.1
     # via -r requirements.in
-Flask-BasicAuth==0.2.0
-    # via -r requirements.in
 gds-metrics @ git+https://github.com/alphagov/gds_metrics_python.git@6f1840a57b6fb1ee40b7e84f2f18ec229de8aa72
```

Comment:
These changes were written by pip-compile, as a result of running make freeze-requirements after the notifications-utils url change in requirements.in, not sure why the reordering, but hopefully we all get identical local results from freeze-requirements and we can keep this